I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -1541,6 +1563,68 @@ Line |     .  +-----+ | |  *  *  | | +-----+  .     | Line
   
   </section>
 
+  <section title="Protection Architectures" anchor="Prot">
+  
+  <t>
+   The YANG model defined in this document supports the following
+   protection architectures:
+  </t>
+    
+  <t><list style="symbols"><?rfc subcompact="yes"?>
+   <t>End-to-end OTSi protection</t>
```

Comment:
If the OT is external, this mechanism is modelled as a segmented protection

I would just call this as "OTSi protection" or re-use the definition in Appendix III/G:873.1 and call it "Individual OTSi(G) protection"

Reply:
Good catch Italo: as depicted in slide 13-14 our model of protection is starting from the port of the splitter on WDM-TE node hosting also the ROADM. So it is a segment protection.

Reply:
I incorporated the comment and will do another commit.